src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,92 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  let totalPages = 0;
+
+  if (perPage !== 0) {
+    totalPages = Math.ceil(total / perPage);
+  }
```

Comment:
Consider handling the case where `perPage` is zero by throwing an error or providing a default value. As it stands, the code silently fails by setting `totalPages` to 0, which might not be the intended behavior if `perPage` is mistakenly set to 0.